it/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,51 +33,51 @@
+xWantToSavePasswordObscured? = Vuoi ricordare il tuo nome utente e la tua password?##I valori archiviati non sono protetti da password.
+xWantToSavePasswordEncrypted? = Vuoi ricordare il tuo nome utente e la tua password?##I valori archiviati non sono protetti da password.
 #translator's note: # in following line will be converted to linefeed by program
-firstPassword = Una Master Password è utilizzata per proteggere i dati sensibili registrati sul vostro disco fisso.##Le prossime versioni utilizzeranno un algoritmo di encrittazione evoluto.#(Lasciate in bianco i campi per non utilizzare una Master Password).#
-password = Inserite la Master Password.
+firstPassword = Si utilizza una Password principale per proteggere i dati sensibili Single Sign-on archiviati sul tuo disco fisso.##Le prossime versioni utilizzeranno un algoritmo di crittografia evoluto.#(Se non si desidera utilizzare una Password principale non compilare i campi).#
+password = Inserisci la tua Password principale.
 #translator's note: # in following line will be converted to linefeed by program
-newPassword = Selezionate una nuova Master Password.#(Lasciate in bianco i campi per non utilizzare una Master Password).#
-enterPassword = Inserite una Master Password
-confirmPassword = Re-inserite la Master Password
-confirmFailed_TryAgain? = The two master passwords were not identical.  Please enter the same master password in both fields.
-SelectUser = Selezionate un nome utente da inserire in questa form.
-SelectUserWhosePasswordIsBeingChanged = Selezionate l'utente al quale modificare la password.
-PasswordNotification = Per comodità, il browser può ricordare i vostri dati di login in modo che non dobbiate reinserirli in futuro. La password sarà oscurata prima di essere salvata sul disco. Volete abilitare questa funzione?
-WalletNotification = Potete salvare i dati inseriti nelle form in modo che vengano inseriti automaticamente in futuro. Per salvare queste informazioni selezionate Modifica/Registra Dati Form dal menu mentre visualizzate la form.
-noPrefills = Non ci sono campi da riempire automaticamente.
+newPassword = Seleziona una nuova Password principale.#(Se non desideri utilizzare una Password principale lasciare i campi in bianco).#
+enterPassword = Inserisci una Password principale
+confirmPassword = Digita di nuovo la Password principale
+confirmFailed_TryAgain? = Le due Password non corrispondono.  Si prega di inserire la stessa Password principale in entrambi i campi.
+SelectUser = Seleziona un nome utente da inserire in questo modulo.
+SelectUserWhosePasswordIsBeingChanged = Seleziona l'utente al quale modificare la password.
+PasswordNotification = Per comodità, il browser può ricordare il tuo nome utente e la tua password, così non dovrai inserirli di nuovo in futuro. La tua password sarà oscurata prima di essere salvata sul tuo disco fisso. Vuoi abilitare questa funzione?
```

Comment:
CORRECTIONS NEEDED. The use of the possessive "tuo", "tua" is redundant. I suggest to modify the sentence as follows: 

Per comodità, il browser può ricordare i tuoi nome utente e password, così non dovrai inserirli di nuovo in futuro. La tua password sarà oscurata prima di essere salvata sul disco fisso. Vuoi abilitare questa funzione?